libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation/B2C/B2COptions.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace EnergyOrigin.TokenValidation.b2c;
+
+public class B2COptions
+{
+    public const string Prefix = "B2C";
+
+    [Required]
+    public string B2CWellKnownUrl { get; init; } = null!;
```

Comment:
Maybe hardcode defaults for now, so we can test quicker? so we have other issue to create them in EO-BASE?

Reply:
Don't we include appsettings files in deployment? (The subsystem using token validation has the configuration in appsettings)

Reply:
You will need to add environment variables to the deployment, in eo-base :)